packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -2,29 +2,34 @@ import React, { PureComponent } from 'react'
 import { Form, Field as FinalFormField } from 'react-final-form'
 
 import Button from 'cozy-ui/react/Button'
+import { translate, extend } from 'cozy-ui/react/I18n'
 import Field from 'cozy-ui/react/Field'
 
 import Manifest from '../Manifest'
 
 export class AccountField extends PureComponent {
   render() {
-    const { type } = this.props
+    const { name, t, type } = this.props
+    const label = t(`fields.${name}.label`)
     const fieldProps = {
+      ...this.props,
       className: 'u-m-0', // 0 margin
+      id: name,
```

Comment:
Are we sure the field name will be unique at this point of code?

Reply:
We're not, but it's not a priority at this point. We should even throw a kind of `ManifestError` if it's the case.

Reply:
name need to be unique

Reply:
Actually, as `fields` is a JS object and name is extracted from key, name will always be unique at this point.

By the way I removed this line.